smithy-templates.json

```diff
@@ -0,0 +1,125 @@
+{
```

Comment:
Should each description say "Uses Gradle to"? Or is it apparent from the repo name?

Reply:
I think this is apparent for the repo name.